lib/DDGC/Web/Service/Blog.pm

```diff
@@ -279,12 +280,44 @@ get '/admin/post/raw/:id' => sub {
     forward '/admin/post/raw', { params('route') };
 };
 
+# TODO: Deprecate this
+sub _add_blog_post_notification_bits {
+    my ( $post ) = @_;
+    my $dbh = schema->storage->dbh;
+    my $now = schema->storage->datetime_parser->format_datetime(DateTime->now);
+    use DDP; p $now;
+
+    my $cycle = $dbh->selectrow_hashref(
+        'SELECT cycle from user_notification
+         WHERE  users_id = ?
+         AND    context_id is NULL
+         AND    user_notification_group_id = 2', {},
```

Comment:
Yup, but you might have trouble slurping an array, then a hash from params like that - this way is more robust I think.